multiversx_sdk_cli/contracts.py

```diff
@@ -62,7 +62,7 @@ class IContractQueryResponse(Protocol):
 
 
 class SmartContract:
-    def __init__(self, address: Optional[Address] = None, bytecode=None, metadata=None):
+    def __init__(self, address: Optional[IAddress] = EmptyAddress(), bytecode=None, metadata=None):
```

Comment:
I think `SmartContract` is the next in line, to be removed (replaced).

Reply:
I agree. I don't know if we should do it now or wait for the sdk to be compliant with the specs.